powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/steps/CreateTokenStep.java

```diff
@@ -130,7 +163,9 @@ public JSONObject execute(JsonStepLogger stepLogger, Map<String, Object> context
         String signatureBaseString = PowerAuthHttpBody.getSignatureBaseString("POST", "/pa/token/create", pa_nonce, requestBytes) + "&" + model.getApplicationSecret();
         String pa_signature = signature.signatureForData(signatureBaseString.getBytes("UTF-8"), Arrays.asList(signaturePossessionKey, signatureKnowledgeKey), counter);
         PowerAuthSignatureHttpHeader header = new PowerAuthSignatureHttpHeader(activationId, model.getApplicationKey(), pa_signature, model.getSignatureType().toString(), BaseEncoding.base64().encode(pa_nonce), model.getVersion());
-        String httpAuhtorizationHeader = header.buildHttpHeader();
+        String httpAuthorizationHeader = header.buildHttpHeader();
+        PowerAuthEncryptionHttpHeader encHeader = new PowerAuthEncryptionHttpHeader(model.getApplicationKey(), activationId, model.getVersion());
+        String httpEncryptionHeader = encHeader.buildHttpHeader();
```

Comment:
I have an issue with the fact there are two HTTP headers and both of them carry the activation ID. This can have bad consequences on the server side...

Reply:
I handled this case on server side by checking that the activation IDs match for both headers, otherwise the endpoint fails with an error. However I agree having two activation IDs is problematic.

Reply:
As agreed, we will only use the signature HTTP header for this case, because all both `activationId` and `applicationKey` is available in the signature HTTP header, too.